apps/bas/package.json

```diff
@@ -86,4 +86,4 @@
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.10.0"
   }
-}
+}
```

Comment:
not sure it really matters but looks like you removed the newline at the end (this is in all of the package.jsons)

Reply:
It's functionally the same. I think I ran `prettier` on things at some point, which might have done that.